src/components/Card.js

```diff
@@ -34,14 +50,48 @@ class Card {
     this._deleteButton = this._cardElement.querySelector(
       ".card__delete-button"
     );
+    this._likeNumber = this._cardElement.querySelector(".card__like-number");
 
     this._setEventListeners();
 
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._name;
     this._cardTitleEl.textContent = this._name;
+
+    if (this._owner._id != this._userID) {
+      this._deleteButton.remove();
+    }
+
+    this.displayCardLike(this._cardLike);
+
     return this._cardElement;
   }
+
+  removeCard() {
+    this._cardElement.remove();
+  }
+
+  updateCardLike(cardLike) {
+    this._cardLike = cardLike;
+    this.displayCardLike(cardLike);
+  }
+
+  displayCardLike(cardLike) {
+    this._likeNumber.textContent = cardLike.length;
+    if (cardLike.some((cardLike) => cardLike._id === this._userID)) {
```

Comment:
Each function should perform just one task, so this code should be moved to a separate method `_isLiked`, which will check if there is a like from the user